.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 2 * * *'
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        rust: ["stable", "beta", "nightly"]
```

Comment:
Can we also test the MSRV? Do we even know what it is?

Reply:
AFAIK, graphgl-client does not yet specify MSRV.